src/components/OAuthLoginButton.tsx

```diff
@@ -24,12 +36,62 @@ export const OAuthLoginButton: FC<OAuthLoginButtonParams> = ({
   }, [login, onSuccess, onFail]);
 
   return (
-    <TouchableOpacity
-      testID={tID('oauth-login-button')}
-      {...touchableOpacityProps}
-      onPress={_login}
-    >
-      {children}
-    </TouchableOpacity>
+    <View style={styles.button}>
+      <TouchableOpacity
+        testID={tID('oauth-login-button')}
+        {...touchableOpacityProps}
+        onPress={_login}
+        style={styles.touchableOpacity}
+      >
+        <View style={styles.content}>
+          <Text style={styles.label}>{label}</Text>
+        </View>
+      </TouchableOpacity>
+    </View>
   );
 };
+
+const defaultStyles = (theme: Theme) => {
+  const button: ViewStyle = {
+    flexDirection: 'row',
+  };
+  const touchableOpacity: ViewStyle = {
+    alignItems: 'center',
+    backgroundColor: theme.colors.secondary,
+    borderRadius: 5 * theme.roundness,
+
+    minWidth: 64,
+  };
+
+  const content: ViewStyle = {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+  };
+
+  const label: TextStyle = {
+    color: theme.colors.onPrimary,
+    marginVertical: 10,
+    marginHorizontal: 24,
+
+    fontFamily: Platform.select({
+      web: 'Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif',
+      ios: 'System',
+      default: 'sans-serif-medium',
+    }),
+
+    fontWeight: '500',
+    letterSpacing: 0.1,
+    lineHeight: 20,
+    fontSize: 14,
+  };
+
+  return { button, touchableOpacity, content, label };
```

Comment:
Instead of declaring variables for each named style we could do something like:

```
return {
  button: {
    // ...
  } as ViewStyle,
  label: {
    // ...
  } as TextStyle,
```

Reply:
By casting, wouldn't we lose the type checking in the object itself?  My understanding was to not prefer casting.  For instance, this compiles just fine:
```
content = {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    notValid: 'garbage',
  } as ViewStyle;
```

Reply:
Great point! We shouldn't do that then.